vagrant/provisioning/roles/confluent-platform-install/tasks/main.yml

```diff
@@ -0,0 +1,46 @@
+---
+# tasks file for confluent-install
+- name: Install curl and which
+  yum:
+    name: "{{ packages }}"
+  vars:
```

Comment:
don't forget to set default values of the new facts here...

Reply:
@david-oc-miller with the last commit i've already defined the variables and their default values in the following facts file: arkcase-ce-facts.yml and arkcase-dev-facts.yml.
I hope it's a proper way, please check

Reply:
@kikomanaskov333 there are many, many other facts files in our Ansible Tower server (one per environment).  To keep from having to update all these other facts files (which do not live here in GitHub) we need to define default values here.  Ansible has a default filter you can use like so: {{ confluent_version | default("5.2") }}, that should do it.  See here: https://docs.ansible.com/ansible/latest/user_guide/playbooks_filters.html#defaulting-undefined-variables